src/library.js

```diff
@@ -720,6 +720,8 @@ LibraryManager.library = {
     return limit;
   },
 
+#if SHRINK_LEVEL < 2 // In -Oz builds, we replace memcpy() altogether with a JS variant, so we should never emit emscripten_memcpy_big() in the build.
```

Comment:
Should this be SHRINK_LEVEL > 2 ?    I would have thought higher SHRINK_LEVEL would mean more compression.

Is there any advantage to doing this?   Won't this only be included on demand anyway?

Reply:
No, in `SHRINK_LEVEL >= 2`, this function is not used, but instead the Wasm side function is used. (we never call out to JS in `SHRINK_LEVEL >= 2`)

Reply:
So I'm a little confused by the comment then .. it says `In -Oz builds, we replace memcpy() altogether with a JS variant,` but that contractions "we never call out to JS in SHRINK_LEVEL >= 2"

Should the comment say `we replace memcpy() altogether with a native variant` ?

Reply:
Good catch, the comment was outdated - fixed now.